src/components/controls/km/km-validation-errors.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div class="scbd-controls km-validation-errors document-validation">
+    <div
+      v-if="reports && reports.isAnalyzing"
+      class="alert alert-info"
+      role="alert"
+    >
+      <i class="bi bi-gear"></i> {{ t("analyzingDocument") }}
+    </div>
+    <div
+      v-if="reports && reports.isSaving"
+      class="alert alert-info"
+      role="alert"
+    >
+      <i class="bi bi-gear"></i> {{ t("savingDocument") }}
+    </div>
+    <div
+      v-if="
+        reports && !reports.isAnalyzing && !reports.isSaving && !reports.errors
+      "
+      class="alert alert-success"
+      role="alert"
+    >
+      {{ t("valid") }}
+    </div>
+    {{ reports }} --- {{ internalReports }}
+    <div v-if="reports && !reports.isAnalyzing && !reports.isSaving">
+      <div
+        v-if="!hideErrors && reports && reports.errors && reports.errors.length"
+        class="alert alert-warning"
+        role="alert"
+      >
+        <button
+          type="button"
+          @click="onButtonClose"
+          aria-label="Close"
+          class="
+            position-absolute
+            top-0
+            end-0
+            m-2
+            bg-transparent
+            border-0
+            fs-5
+            text-secondary
+          "
+        >
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <strong>
+          {{ t("contains") }} {{ reports.errors.length }} {{ t("errors") }}
```

Comment:
use i18n plural https://vue-i18n.intlify.dev/guide/essentials/pluralization

Reply:
Used i18n plural "stub" in the existing composable.